Sources/Starknet/Data/Felt.swift

```diff
@@ -52,6 +55,21 @@ extension Felt: Codable {
     }
 }
 
+extension Felt: AdditiveArithmetic {
+    public static func - (lhs: Felt, rhs: Felt) -> Felt {
+        if (rhs.value > lhs.value) {
+            let value = prime - (rhs.value - lhs.value)
+            return Felt(value)!
+        }
+        
+        return Felt(lhs.value - rhs.value)!
+    }
+    
+    public static func + (lhs: Felt, rhs: Felt) -> Felt {
+        return Felt((lhs.value + rhs.value).quotientAndRemainder(dividingBy: prime).remainder)!
+    }
+}
```

Comment:
Are you sure we want to expose felt arithmetic? I think they could be source of many bugs.

Reply:
In jvm sdk we don't offer those on purpose. In most cases you want to do arithmetic operations not over a finite field.

Reply:
Makes sense, as the underlying BigUInt value is public, let's stick to using that for calculation